pkg/cli/admin/release/codesign/codesign.go

```diff
@@ -0,0 +1,272 @@
+// Copyright 2020 The Go Authors. All rights reserved.
```

Comment:
Nit: can you move this under `pkg/cli/admin/internal/codesign` pacakge, this will ensure that nobody consuming oc will ever use it. Especially that we're copying golang code.